modules/oc-mirror-enclave-support-about.adoc

```diff
@@ -19,4 +19,36 @@ Enclave support has the following benefits:
 
 * You can mirror content directly from the centralized internal registry to enclaves without restarting the mirroring process from the internet for each enclave.
 
-* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
\ No newline at end of file
+* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
+
+[id="oc-mirror-enclave-how-to_{context}"]
+== Enclave mirroring workflow
+
+image::445_OpenShift_Enclave_support_0724.png[Enclave Support]
+
+The previous image outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.
+
+*Environment with Internet Connection*:
+
+. The user executes oc-mirror plugin v2 to mirror content from an online registry to a local disk directory.
+. The mirrored content is saved to the disk for transfer to offline environments.
+
+*Disconnected Enterprise Environment (No Internet)*:
+
+*  Flow 1:
+
+** The user runs oc-mirror plugin v2 to load the mirrored content from the disk directory, which was transferred from the online environment, into the `enterprise-registry.in` registry.
+
+* Flow 2:
+
+.. After updating the `registries.conf` file, the user executes the oc-mirror plugin v2 to mirror content from the `enterprise-registry.in` registry to an enclave environment.
+
+.. The content is saved to a disk directory for transfer to the enclave.
+
+*Enclave Environment (No Internet)*:
+
+. The user runs oc-mirror plugin v2 to load content from the disk directory into the `enclave-registry.in` registry.
```

Comment:
It was good to change this from the hyphen format, but I think that because this list has only a single item, there is no possible order to follow, in which case this can be an unordered bullet:
```suggestion
* The user runs oc-mirror plugin v2 to load content from the disk directory into the `enclave-registry.in` registry.
```